### PR TITLE
Add new services

### PR DESCRIPTION
**Pull request by duncanjbrown**
- Teacher pay calculator
- Refer a teacher for serious misconduct
- Check the Children's barred list
- Access your teaching qualifications
- Get a teacher relocation payment

Also stop docs breaking builds when they have non-breaking spaces in the title

### CODE REVIEW

File: config/services.yml [hunk 1]
```diff
@@ -143,3 +143,27 @@
   csproj_path: 'GetIntoTeachingApi/GetIntoTeachingApi.csproj'
   docsets:
     - path: 'docs'
+- name: "Teacher pay calculator"
+  repo_name: "DFE-Digital/teacher-pay-calculator"
+  language: 'ruby'
+  docsets: []
+- name: "Refer a teacher for serious misconduct"
```
Comment (gpeng) [inline config/services.yml:150]: This one is already on 120 with a slightly different name. I think the existing entry is correct.

File: config/services.yml [hunk 2]
```diff
@@ -143,3 +143,27 @@
...
+  repo_name: "DFE-Digital/refer-serious-misconduct"
+  language: 'ruby'
+  docsets:
+    - path: 'docs'
+- name: "Check the Children's barred list"
+  repo_name: "DFE-Digital/check-childrens-barred-list"
+  language: 'ruby'
+  docsets:
+    - path: 'docs'
+- name: "Access your teaching qualifications"
```
Comment (gpeng) [inline config/services.yml:160]: This one is on 114.

**Review by gpeng — COMMENTED** [review]
Thanks for doing this @duncanjbrown. Just a couple of comments. They seem to be in the list already but aren't showing on the deployed site 🤷  For 'Access your teaching qualifications' (and I suppose the same applies for Apply for ITT and Publish) I wonder is there any value in us being able to list the other service names that are in those repos. (I don't mean in this PR, just thinking aloud).

**Comment by duncanjbrown** [comment]
Superseded by #48

**Pull request closed without merging**